central/authprovider/datastore/datastore_impl.go

```diff
@@ -57,7 +57,8 @@ func (b *datastoreImpl) UpdateAuthProvider(ctx context.Context, authProvider *st
 	b.lock.Lock()
 	defer b.lock.Unlock()
 
-	if err := b.verifyExistsAndMutable(ctx, authProvider.GetId(), force); err != nil {
+	// The API currently does not allow setting the force flag since it would require a non-compatible gRPC change.
```

Comment:
This is datastore code. What the API allows or does not allow is of secondary importance for this code.

I would suggest to rephrase this as something like

```
// Currently, the data store does not support forcing updates. If we want to add a force flag to the respective API methods, we might need to revisit this.
```